src/Model/ContentItemContainer.php

```diff
@@ -24,7 +24,7 @@ public function getContentItemMatrix();
      * Returns a list of ContentItems
      *
      * @param string|null $region
-     * @return iterable<int, ContentItem>
+     * @return iterable<int, ContentItem>|list<ContentItem>
```

Comment:
`list<ContentItem>` is al een afgeleide van `iterable<int, ContentItem>`. Een `list` is een `iterable` met indexed (int) keys

Reply:
Ja, maar deze exacte toevoeging is wel nodig om de errors die Psalm geeft in de Cookiecutter PR op te lossen